index.html

```diff
@@ -1,325 +0,0 @@
-                             <img src="images/sidebar/chibirdart 1.png" alt="">
-                             <div class="name-profile-side-bar">
-                                <h1>chibirdart</h1>
-                                <h2>Segue você</h2>
-                            </div>
-                        </div>
-                        <div class="right-profile-body-side-bar">
-                            <h1>Seguir</h1>
-                        </div>
-                    </div>
-                    <div class="profile-body-side-bar">
-                        <div class="left-profile-body-side-bar">
-                            <img src="images/sidebar/razoesparaacreditar 1.png" alt="">
-                            <div class="name-profile-side-bar">
-                               <h1>razoesparaacreditar</h1>
-                               <h2>Segue você</h2>
-                           </div>
-                       </div>
-                       <div class="right-profile-body-side-bar">
-                           <h1>Seguir</h1>
-                       </div>
-                   </div>
-                   <div class="profile-body-side-bar">
-                         <div class="left-profile-body-side-bar">
-                             <img src="images/sidebar/smallcutecats 1.png" alt="">
-                             <div class="name-profile-side-bar">
-                                <h1>smallcutecats</h1>
-                                <h2>Segue você</h2>
-                            </div>
-                        </div>
-                        <div class="right-profile-body-side-bar">
-                            <h1>Seguir</h1>
-                        </div>
-                    </div>
-
-                   
-                </div>
-                <div class="footer-side-bar">
-                    <h1>
-                        Sobre • Ajuda • Imprensa • API • Carreiras • Privacidade •
-                        Termos • Localizações • Contas mais relevantes • Hashtags • 
-                        Idioma
-                    </h1>
-                    <h2>
-                        © 2021 INSTAGRAM DO FACEBOOK
-                    </h2>
-                    
-                </div>
-            </div>
-       
-            </div>
-    </div>
-    <div class="mobile-rodape">
-        <div class="ricons">
-          <ion-icon name="home"></ion-icon>
-        </div>
-        <div class="ricons">
-          <ion-icon name="search-outline"></ion-icon>
-        </div>
-        <div class="ricons">
-          <ion-icon name="add-circle-outline"></ion-icon>
-        </div>
-        <div class="ricons">
-          <ion-icon name="heart-outline"></ion-icon>
-        </div>
-        <div class="ricons">
-          <ion-icon name="person-outline"></ion-icon>
-        </div>
-        <script
-          type="module"
-          src="https://unpkg.com/ionicons@5.4.0/dist/ionicons/ionicons.esm.js"
-      ></script>
```

Comment:
Opaaa, seria interessante colocar esse script no head, mas por quê?

Bem, é atraves desse script que estamos tendo acesso aos icones. Então colocando ele no head a gente garante que o script seja lido e executado antes de qualquer elemento na pagina, dessa forma voce garante que ele estará presente quando a página for "montada".